packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+      }
+    }
+  ],
+  "_links": {
+    "next": {
+      "href": "http://${yourOktaDomain}/api/v1/iam/resource-sets?after=iamoJDFKaJxGIr0oamd0q"
+    }
+  }
+}
+```
+
+### Update resource set
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="put" url="/api/v1/iam/resource-sets/${resourceSetId}" />
+
+Update label and description of a Resource Set
+
+#### Request parameters
+
+| Parameter     | Description                               | Param Type   | DataType     | Required |
+| :------------ | :---------------------------------------- | :----------- | :----------- | :------- |
+| resourceSetId | Unique id of the Resource Set             | URL          | String       | TRUE     |
+| label         | New unique name given to the Resource Set | Body         | String       | TRUE     |
+| description   | New description of the the Resource Set   | Body         | String       | TRUE     |
+
+#### Response parameters
+
+Updated [Resource Set](#resource-set-object)
+
+#### Request example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+      "label": "SF-IT-Staff",
+      "description": "Staff in the IT department of San Francisco"
+    }' "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+```
+
+#### Response example
+
+```json
+{
+  "id": "iamoJDFKaJxGIr0oamd9g",
+  "label": "SF-IT-Staff",
+  "description": "Staff in the IT department of San Francisco",
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    },
+    "resources": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/resources"
+    },
+    "bindings": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+    }
+  }
+}
+```
+
+### Delete resource set
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetId}" />
+
+Deletes a Resource Set and all its associated Bindings.
+
+#### Request parameters
+
+| Parameter      | Description                          | Param Type   | DataType                              | Required |
+| :------------- | :----------------------------------- | :----------- | :------------------------------------ | :------- |
+| resourceSetId  | `id` of the Resource Set             | URL          | String                                | TRUE     |
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+### Resource operations
+These help adding, removing and listing of resources within a Resource Set
```

Comment:
Reword sentence ending with a period > These operations add, remove, and list the resources within a Resource Set.